aderyn_core/src/audit/auditor.rs

```diff
@@ -17,5 +27,31 @@ pub trait AuditorDetector: Send + Sync + 'static {
         String::from("Title")
     }
 
-    fn print(&self, _context: &WorkspaceContext) {}
+    fn instances(&self) -> Vec<AuditorInstance>;
+}
+
+pub trait AuditorPrinter {
```

Comment:
Hey @TilakMaddy , I think you'll need to rethink this for multiple workspaces. Each AuditorDetector will have its own print method, because every single one will have bespoke output and be formatted differently.

I'm reintroducing the print method in a separate PR after getting more input from the audit team. Stay tuned!

Reply:
#445

Reply:
I mean so long as it is a table, we can make changes to the auditor instances to be any string array (Instead of specific fields that we have now)

Reply:
Why don't we take it up after merging the current ICF to dev (or the equivalent - like adding process_foundry.rs, etc)?

Reply:
I know how to resolve this one now :) I'll throw up some changes for dev shortly